tests/default_vars.sh

```diff
@@ -22,9 +22,13 @@ elif [[ $MACHINE_ID = wcoss_dell_p3 || $MACHINE_ID = wcoss2 ]]; then
   TASKS_strnest=96 ; TPN_strnest=28 ; INPES_strnest=2 ; JNPES_strnest=4
 
   TASKS_cpl_dflt=192; TPN_cpl_dflt=28; INPES_cpl_dflt=3; JNPES_cpl_dflt=8
-  THRD_cpl_dflt=1; WPG_cpl_dflt=6;  MPB_cpl_dflt="0 143"; APB_cpl_dflt="0 149"
+  THRD_cpl_dflt=1; WPG_cpl_dflt=6;  ePB_cpl_dflt="0 143"; APB_cpl_dflt="0 149"
```

Comment:
You have a typo here and elsewhere: ePB_cpl_dflt. 

The WPG is the atm write component tasks, the MPB is the mediator, the APB is the atmosphere, the OPB is the ocean and the IPB is the ice.